.pre-commit-config.yaml

```diff
@@ -1,4 +1,51 @@
+---
 repos:
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
+        args:
```

Comment:
Is it possible to add these args as config in pyproject.toml? Looks good otherwise!

Reply:
Can always add "move some of the pre-commit config to pyproject" as a to-do item.

Reply:
https://github.com/glass-dev/glass/issues/267

Reply:
Sorry this is my bad. These options have been tried and tested, but I have some hacky "global" pre-commit hooks, where I don't want to have the same config everywhere https://github.com/paddyroddy/.github/blob/main/precommit/general/general-hooks.yaml

Reply:
Ah, thanks for looking into this, keeping it as it is should be fine!